src/System.Management.Automation/engine/runtime/Operations/MiscOps.cs

```diff
@@ -1610,6 +1610,13 @@ internal static void CheckActionPreference(FunctionContext funcContext, Exceptio
                 preference = ExceptionHandlingOps.QueryForAction(rte, rte.Message, context);
             }
 
+            // if the error action preference is set to break, then break
+            // immediately into the debugger
```

Comment:
Could you please explain what tool you use for 80 column auto word wrapping and why? It seems that for modern displays 120-140 width is very good. (We could adjust config files in our repo)

Reply:
I don't use a tool for auto-word wrapping of comments. I just inspected the wrapping that I already saw in the files/projects I was working in, and eyeballed it, making my comments wrap the same way. Working with so many different projects over the years, I've become accustomed to following whatever style is already in place, so that the file/project is consistent. If we want comment wrapping to be at a certain column width instead, I'm happy to switch to whatever we want to use.